FreshRSS/FreshRSS
# Article css filtering

lets you add comma separated CSS selectors that will be removed from the extracted article content. Preview, import, and export work too. Let me know what else I can do to make this PR better

How to test the feature manually:

1. Add a feed with CSS selector and filtering to make sure the filtered elements are removed
2. Add a feed with CSS selector and no filtering to make sure plain old filtering works
3. Preview a feed via the css selector to make sure the css filters are removed in the preview
4. Export and import to make sure filtering OPML elements work correctly

Pull request checklist:
